Sources/TensorFlow/Loss.swift

```diff
@@ -24,6 +24,23 @@ public func meanSquaredError<Scalar: TensorFlowFloatingPoint>(
     return (expected - predicted).squared().mean()
 }
 
+/// Computes the mean squared logarithmic error between predictions and expectations.
```

Comment:
```suggestion
/// Returns the mean squared logarithmic error between predictions and expectations.
```

See #160 for more context.